src/script/ismine.cpp

```diff
@@ -28,6 +28,14 @@ enum class IsMineSigVersion
     WITNESS_V0 = 2  //! P2WSH witness script execution
 };
 
+enum class IsMineResult
+{
+    NO = 0,
+    WATCH_ONLY = 1,
+    SPENDABLE = 2,
+    INVALID = 3,
```

Comment:
nit, add comment to note that the value matters and `INVALID` must be the higher?

Reply:
Done, added some comments.